src/config.ts

```diff
@@ -92,6 +92,7 @@ export default {
       'brother-sister',
       'brothers-sisters',
       'daft',
+      'easy',
       'master',
```

Comment:
could we also add `'easily'`?

Reply:
Added the word 'easily' to the ignore list too.

Reply:
`easily` is covered by `easy`